tests/drivers/flash/common/Kconfig

```
@@ -4,11 +4,15 @@
 # Device/scenario dependent information that is not available in
 # other ways.
 
+DT_CHOSEN_Z_FLASH := zephyr,flash
+
 config TEST_DRIVER_FLASH_SIZE
 	int "Size of flash device under test"
-	default -1
+	default -1 if !SOC_FAMILY_STM32
+	default $(dt_chosen_reg_size_int,$(DT_CHOSEN_Z_FLASH),0) if SOC_FAMILY_STM32
```

Oh, no! That what I have meant:
```suggestion	
	default $(dt_chosen_reg_size_int,$(DT_CHOSEN_Z_FLASH),0) if SOC_FAMILY_STM32
	default -1
```
Sorry for confusion.

---

Ok, I see
Done